discordbot/requirements.txt

```diff
@@ -1,4 +1,4 @@
-aiohttp==3.7.4.post0; python_full_version >= "3.5.3" and python_version >= "3.6"
+aiohttp==3.7.4.post0; python_version >= "3.6" and python_full_version >= "3.5.3"
```

Comment:
Did you intended to change this file?

Reply:
Humm, if it was a change when you merged with main it shouldn't have shown any difference, since the diff compares with main.

I don't think you added any new package here, right? If so, personally, I would avoid updating the requirements since the dependencies can get quite tricky to fix. I always try to avoid touching these files as much as I can.

Reply:
Yes you are right. Pycoingecko is not used on discordbot yet so this should not be changed. Reverted 👍